scrapers/base.py

```diff
@@ -0,0 +1,68 @@
+import boto3
+import json
+import logging
+import os
+import requests
+
+
+from bs4 import BeautifulSoup
+from time import gmtime, strftime
+
+
+from .config import AWS
+
+
+class Scraper(object):
+	"""
+	This is a base class inherited by other scraper classes.
+	"""
+
+	def __init__(self):
+		self.url = None
+		self.s3 = boto3.client("s3", **{
+			"aws_access_key_id": AWS["aws_access_key_id"],
+			"aws_secret_access_key": AWS["aws_secret_access_key"],
+			"region_name": AWS["region_name"]
+		})
+
+	def get_html_content(self, url):
+		"""This method returns the entire html content of a page."""
+		try:
+			res = requests.get(url)
+			html = BeautifulSoup(res.content, 'html.parser')
+			return html 
+		
+		except Exception as err:
+			print "Url is not reachable. Error logged on: {}".format(
+				strftime("%Y-%m-%d %H:%M:%S", gmtime()))
+			logging.exception(str(err))	   
+		
+	def scrape(self, url, base_url):
+
+		"""This method is used as a sraper for 
+		the website url's that are passed from it."""
+
+		result_html = self.get_html_content(url)
+		data = []
+		if result_html:
+			ul = result_html.find("ul", class_="business-lhs")
```

Comment:
@andela-amutava Nice work. You may consider refactoring this function as this block it will change from site to site. Also in the event of site redesign, you may want to catch that too.